apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -66,7 +71,19 @@ function Navbar() {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
-					<Button text="Login" href="/login" isLightVersion />
+					{!status && <NavLinkItem href="/apply">Apply</NavLinkItem>}
+					{status !== null && (
+						<NavLinkItem href="/portal">Portal</NavLinkItem>
+					)}
+					{uid === null ? (
+						<Button text="Login" href="/login" isLightVersion />
+					) : (
+						<Button
+							text="Logout"
+							href="/api/user/logout"
```

Comment:
Issue: `Button` uses next/link under the hood which will [prefetch the link](https://nextjs.org/docs/app/api-reference/components/link#prefetch) causing the browser to load the logout page.

Reply:
Could we also add a rewrite or separate handler for `/logout` so the end user won't see the API path on link hover?

Reply:
Think the Next server-side redirect or rewrite could also fix the no-refresh issue.

Reply:
Got it to work, but I had to sacrifice `next/link` in the process.